docs/cpu_templates/cpu-templates.md

```diff
@@ -152,6 +152,12 @@ If a custom CPU template makes modifications to a CPUID leaf that is
 not reported as supported by KVM via `KVM_GET_SUPPORTED_CPUID`,
 the default register values of such a leaf are assumed to be `0`.
 
+#### CPUID normalization
```

Comment:
It's worth to have documents about MSR modification and ARM register modification.
https://github.com/firecracker-microvm/firecracker/blob/feature/cpu-templates/src/vmm/src/vstate/vcpu/x86_64.rs#L239-L242
https://github.com/firecracker-microvm/firecracker/blob/feature/cpu-templates/src/vmm/src/vstate/vcpu/aarch64.rs#L110-L116

Reply:
nice, added!